components/tabs/demo/link-router.ts

```diff
@@ -0,0 +1,34 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'nz-demo-tabs-link-router',
+  template: `
+    <nz-tabset nzLinkRouter>
+      <nz-tab>
+        <span nz-tab-title
+          ><a [routerLink]="['.']" [queryParams]="{ tab: 'one' }" queryParamsHandling="merge">Default</a></span
+        >
+        Default.
+      </nz-tab>
+      <nz-tab>
+        <span nz-tab-title
+          ><a [routerLink]="['.']" [queryParams]="{ tab: 'two' }" queryParamsHandling="merge">Two</a></span
```

Comment:
`<a nz-tab-link [routerLink]="['.']" [queryParams]="{ tab: 'two' }" queryParamsHandling="merge">Two</a>`
is a better API I think